### PR TITLE
Shimmer: change prop use of subcomponents.

### PR DESCRIPTION
**Pull request by Vitalius1**
#### Pull request checklist

- [X] Include a change request file using `$ npm run change`

#### Description of changes
Removing use of `borderStyle` prop of Shimmer subcomponents in favor of using mergeStyles API.

### CODE REVIEW

**Comment by cliffkoh** [comment]
Sorry about the merge conflict. This fixes the copy of Shimmer in OUFR but not the one in experiments.. May I strongly recommend reconciling the fact that they're now forking? Either by removing the duplication or making the changes in both places.

Also, it might be good to add an example using the styles API to demonstrate how to customize the borderStyle (and to verify that it still works) :)

File: packages/office-ui-fabric-react/src/components/Shimmer/ShimmerGap/ShimmerGap.types.ts [hunk 1]
```diff
@@ -26,11 +26,6 @@ export interface IShimmerGapProps extends React.AllHTMLAttributes<HTMLElement> {
    */
   width?: number | string;
 
-  /**
-   * Sets custom styling of the gap.
-   */
-  borderStyle?: IStyleSet;
```
Comment (cliffkoh) [inline packages/office-ui-fabric-react/src/components/Shimmer/ShimmerGap/ShimmerGap.types.ts]: @dzearing are you okay with removing this prop? It is technically a breaking change... 

An alternative is just to change the type to `any` and add `@deprecated`.  (Would be even nicer if we could use TS 3.0's `unknown` type instead for deprecation).

**Comment by cliffkoh** [comment]
Request review and input from @dzearing as well.

File: packages/office-ui-fabric-react/src/components/Shimmer/ShimmerGap/ShimmerGap.types.ts (hunk 1 above)
Comment (dzearing) [reply packages/office-ui-fabric-react/src/components/Shimmer/ShimmerGap/ShimmerGap.types.ts]: Unless it was a blocking bug, this should be @deprecated and not removed.

**Comment by dzearing** [comment]
@Vitalius1 can you clean up conflicts and deprecate the prop?

File: packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx [hunk 2]
```diff
@@ -62,7 +62,7 @@ export class TooltipHost extends BaseComponent<ITooltipHostProps, ITooltipHostSt
       (tooltipProps && tooltipProps.onRenderContent && tooltipProps.onRenderContent())
     );
     const showTooltip = isTooltipVisible && isContentPresent;
-    const ariaDescribedBy = (setAriaDescribedBy && isTooltipVisible && isContentPresent) ? tooltipId : undefined;
+    const ariaDescribedBy = setAriaDescribedBy && isTooltipVisible && isContentPresent ? tooltipId : undefined;
```
Comment (Vitalius1) [inline packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx]: This line was changed by the auto formatter (prettier) after merging a fresh master and resolving the conflicts.

**Comment by Vitalius1** [comment]
@cliffkoh @dzearing 
Closing this PR due to some snapshots issues I had after some failed git manipulation :cry:
Opening this one instead on a clean branch: [PR 5559](https://github.com/OfficeDev/office-ui-fabric-react/pull/5739).

**Pull request closed without merging**